File: sandbox.client/front/src/model/ClientAddress.ts
import {AddressType} from "./AddressType";

export class ClientAddress {
  constructor(client: number, type: AddressType, street: string, house: string, flat: string) {
    this.client = client;
    this.type = type;
    this.street = street;
    this.house = house;
    this.flat = flat;
  }

  public client: number /* int */;
  public type: AddressType;
  public street: string;
  public house: string;
  public flat: string;
}

File: sandbox.db/src/kz/greetgo/sandbox/db/migration/innerMigration/ClientAddress.java
package kz.greetgo.sandbox.db.migration.innerMigration;

public class ClientAddress {
  public String street, house, flat;

  public ClientAddress(String street, String house, String flat) {
    this.street = street;
    this.house = house;
    this.flat = flat;
  }

  @Override
  public String toString() {
    return "ClientAddress{" +
      "street='" + street + '\'' +
      ", house='" + house + '\'' +
      ", flat='" + flat + '\'' +
      '}';
  }
}
